week-4/Homework/mandatory/2-writers.js

```diff
@@ -54,19 +54,36 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 
-
+writers.forEach(function (person) {
+  console.log(`Hi, my name is ${person.firstName} ${person.lastName}. I am ${person.age} years old, and work as a ${person.occupation}.`);
+})
 
 /*
 Exercise 2:
   Only `console.log()` out the writers who are in their 40s (meaning between 40 and 49)
   and not with us anymore, use the below sentence format:
   "Writer {firstName} {lastName} died at {age} years old."
 */
-
+writers.forEach(function(person) {
+  if (person.age <= 49 && person.age >= 40) {
+    if (person.alive == false) {
+      console.log(`Writer ${person.firstName} ${person.lastName} died at ${person.age} years old.`);
+      
+    }

```

Comment:
Will I need to use parentheses as well if I’m using the unary ! Operator?

If !person.alive OR if (!person.alive) OR if !(person.alive) 

Which way is the right way to use it?

Reply:
Yes, you must use parenthesis on if statement first because it is part of the syntax then everything else should go inside, otherwise, you would get a syntax error. So the middle example is the correct.

Reply:
Thanks Zoltan.